### ISSUE TITLE
Queue breaks if : is in text

### ISSUE THREAD

**Issue by fosspill** [eerotal/LibreSignage#109]
While playing around with the version supplied through libresinage:latest I found a breaking issue...

The whole interface stops responding properly, and the queue with the issue cannot be deleted or fixed. It's just stuck there.

**Steps to reproduce (or, what I did..)**
- Create a new queue (ex: my-test).
- Create another slide and insert:
```
Testing: Test!
```
- Save.. Stuff is now half broken, so a refresh is needed.

**You are no longer able to access or delete the queue AT ALL.**

**Expected Result**
An error, if the content is incorrect.
The ability to correct said error
The ability to delete the broken queue

**Comment by eerotal** [comment]
Yeah, I managed to reproduce this on v1.1.0. I need to take a look at this soon.

**Comment by fosspill** [comment]
1.2 is out now with a good amount of changes! Do you know if the changes may have resolved this issue?

**Comment by eerotal** [comment]
Unfortunately I haven't had the time and motivation to look into this issue yet.
